src/firefly/js/Firefly.js

```diff
@@ -145,7 +145,7 @@ const defFireflyOptions = {
         minScatterGLRows: 1000,         // minimum number of points to use WebGL 'scattergl' instead of SVG 'scatter'
         singleTraceUI: false,           // by default we support multi-trace in UI
         upperLimitUI: false,            // by default user can not set upper limit column in scatter options
-        useChartWorkArea: false,        // true to use Chart container with 'pin' feature
+        allowPinnedCharts: false,        // true to use Chart container with 'pin' feature
```

Comment:
suit.html, suit-test.html irsaviewer.html all use `useChartWorkArea`. You should make sure you change it in all of them.

Reply:
Yes.  They were mentioned under the `More changes here:` section.  You were assigned as reviewer of those PRs as well.

Reply:
i missed that. OK- I approved those as well.